src/Model/FileReader.java

```diff
@@ -0,0 +1,41 @@
+package Model;
+
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+
+public class FileReader {
+
+
+  public FileReader(Object s) {
+
+  }
+
+
+  public static void getFile() throws IOException {
+
+    String fileName = "C:/Users/Krisi/Desktop/resources/mysteryFiles/mystery1.txt";
+
+    try (FileInputStream fis = new FileInputStream(fileName)) {
+
+
+      int i = 0;
+
+      do {
+
+        byte[] buf = new byte[1024];
```

Comment:
make 1024 a constant with an appropriate name, so its easier to read what it does and to be reusable

Reply:
also, do this for all the numbers in this file so we can better understand what they mean